libs/perun/components/src/lib/password-reset/password-reset.component.ts

```diff
@@ -53,8 +55,9 @@ export class PasswordResetComponent implements OnInit {
   }
 
   resetPassword(login: string) {
-    const url = this.store.get('pwd_reset_base_url');
-    location.href = `${url}?login-namespace=${login}`;
+    this.utilsService.getAppsConfig().subscribe(config => {
```

Comment:
you are calling this method every time you need the config, cant the config be stored at something like "StoreService" and load it at the start of the application?

Reply:
Good point, I rewrote it.